### PR TITLE
fix: update query-queue to version 1.0.0

### LINKED ISSUE #202: Integration tests for query-queue issue errors with v1.0.0-0.6.0
passes with v0.16.0-0.5.1: https://github.com/line/cosmwasm/runs/6581567071?check_suite_focus=true

errors with v1.0.0-0.6.0: https://github.com/line/cosmwasm/runs/6684858016?check_suite_focus=true

### PR DESCRIPTION
**Pull request by loloicci**
# Description
- update gas_limit to follow vm v1.0.0
- update how to query raw to follow queue v1.0.0
- change the result format of raw_query

This fixes #202 

## Types of changes

- [x] Bug fix (changes which fixes an issue)
- [ ] New feature (changes which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [x] I followed the [contributing guidelines](https://github.com/line/cosmwasm/blob/main/CONTRIBUTING.md).
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

File: contracts/query-queue/tests/integration.rs [hunk 1]
```diff
@@ -40,7 +40,7 @@ static QUEUE_WASM: &[u8] =
     include_bytes!("../../queue/target/wasm32-unknown-unknown/release/queue.wasm");
 
 fn create_contract() -> (Instance<MockApi, MockStorage, MockQuerier>, MessageInfo) {
-    let gas_limit = 500_000_000; // enough for many executions within one instance
+    let gas_limit = 1_000_000_000_000; // ~1ms, enough for many executions within one instance
```
Comment (zemyblue) [inline contracts/query-queue/tests/integration.rs:43]: Why does the gas increase very big?

File: contracts/query-queue/tests/integration.rs [hunk 2]
```diff
@@ -182,7 +182,7 @@ fn create_integrated_query_contract() -> (Instance<MockApi, MockStorage, MockQue
                     .with_storage(|storage| VmResult::Ok(storage.get(key).0.unwrap()))
                     .unwrap();
                 if data.is_none() {
-                    return SystemResult::Err(SystemError::Unknown {});
+                    return SystemResult::Ok(ContractResult::Ok(Binary::from(b"null")));
```
Comment (zemyblue) [inline contracts/query-queue/tests/integration.rs:185]: Wow! This return value is a bit complicated.

**Comment by zemyblue** [comment]
@loloicci , The CI action of `query_queue` contract still failed.

File: contracts/query-queue/tests/integration.rs (hunk 1 above)
Comment (loloicci) [reply contracts/query-queue/tests/integration.rs:43]: it follows https://github.com/CosmWasm/cosmwasm/pull/1120

File: contracts/query-queue/tests/integration.rs (hunk 2 above)
Comment (loloicci) [reply contracts/query-queue/tests/integration.rs:185]: Before this change, it errors if there is no value in storage bonded with the specified key. (And, it confused me for fixing 😅 )
This is not good for testing, so fix it to return the binary formed string "null".

**Comment by loloicci** [comment]
> The CI action of query_queue contract still failed.
Sorry, it has not included Cargo.lock and schema JSONs. I pushed them.

**Review by zemyblue — APPROVED** [review]
LGTM

**Review by shiki-tak — APPROVED** [review]
LGTM

**Pull request merged**